services/self-service/src/main/java/com/epam/dlab/backendapi/domain/CreateProjectDTO.java

```diff
@@ -15,7 +15,7 @@
 	private final Set<String> groups;
 	@NotNull final Set<String> endpoints;
 	@NotNull
-	@Pattern(regexp = "^ssh-.*\\n?", message = "Wrong key format. Key should be in openSSH format")
+	@Pattern(regexp = "^ssh-.*\\n?", message = "format is incorrect. Please use the openSSH format")
```

Comment:
Do we need space before "format"?

Reply:
No, we don't.
The original error message:
'''
    "code": 400,
    "message": "key format is incorrect. Please use the openSSH format"
'''